blocks/block-alignment-toolbar/index.js

```diff
@@ -4,6 +4,11 @@
 import { __ } from 'i18n';
 import { Toolbar } from 'components';
 
+/**
+ * Internal dependencies
+ */
+import withEditorSettings from '../settings/with-editor-settings';
```

Comment:
Do we need the nesting in `settings`. What else will live there? Maybe just `blocks/with-editor-settings/index.js` ?